### PR TITLE
Add Dialogue Engine

### PR DESCRIPTION
**Pull request by Rubonnek**
The plugin has been gathering a small user base organically through the [asset library](https://godotengine.org/asset-library/asset/2598).

### CODE REVIEW

**Review by Calinou — CHANGES_REQUESTED** [review]
Please follow alphabetical order in that list (it should be between CSLocator and Godot Google Play Service).

**Comment by Rubonnek** [comment]
> Please follow alphabetical order in that list (it should be between CSLocator and Godot Google Play Service).

Done

**Review by Calinou — APPROVED** [review]
Thanks!

File: README.md [hunk 1]
```diff
@@ -253,6 +253,7 @@ See [Vivraan/godot-lang-support](https://github.com/Vivraan/godot-lang-support).
 
 - [AnimatedShape2D](https://github.com/Goutte/godot-addon-animated-shape-2d) - Animate a CollisionShape2D along with the frames of an AnimatedSprite2D.
 - [CSLocator (Contextual Service Locator)](https://codeberg.org/svetogam/cslocator) - Provides a clean interface to register and find objects through the scene tree like localized singletons.
+- [Dialogue Engine](https://github.com/Rubonnek/dialogue-engine) - A minimalistic dialogue engine that fits into your GUI nodes and automatically graphs the branching dialogues for easy debugging.
```
Comment (Calinou) [inline README.md]:
```suggestion
- [Dialogue Engine](https://github.com/Rubonnek/dialogue-engine) - Minimalistic dialogue engine that fits into your GUI nodes and automatically graphs the branching dialogues for easy debugging.
```

**Pull request merged**